annotations/handler.go

```diff
@@ -31,7 +32,15 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer resp.Body.Close()
 
 	w.Header().Add("Content-Type", "application/json")
-	w.WriteHeader(resp.StatusCode)
+	if resp.StatusCode == http.StatusOK || resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusBadRequest {
+		w.WriteHeader(resp.StatusCode)
+		io.Copy(w, resp.Body)
+	} else {
+		writeMessage(w, "Service unavailable", http.StatusServiceUnavailable)
+	}
+}
 
-	io.Copy(w, resp.Body)
+func writeMessage(w http.ResponseWriter, msg string, status int) {
+	w.WriteHeader(status)
+	w.Write([]byte(fmt.Sprintf(`{"message":"%v"}`, msg)))
```

Comment:
OK in this case, but I would have thought better in general to use a `map[string]interface{}` and delegate to a JSON encoder (e.g. in case the message contains quotation marks)